packages/agreement-process/src/services/agreementSubmissionProcessor.ts

```diff
@@ -130,8 +130,7 @@ const submitAgreement = async (
   const newState = agreementStateByFlags(
     nextStateByAttributes,
     undefined,
-    undefined,
-    suspendedByPlatform
```

Comment:
If we want to remove the update of the `suspendedByPlatform` flag also from the submit, as discussed, you should also remove the other places where it is defined and updated in this file:

* `const suspendedByPlatform - ...` a few lines above
* the `suspendedByPlatform` param in `getUpdateSeed`